ksql-engine/src/test/java/io/confluent/ksql/function/udf/url/UrlDecodeKudfTest.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.confluent.ksql.function.udf.url;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import io.confluent.ksql.function.KsqlFunctionException;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import org.junit.Before;
+
+public class UrlDecodeKudfTest {
+
+  private static final String inputEncodedValue = "%3Ffoo+%24bar";
+  private static final String inputSpecialChars = "foo.-*_bar";
```

Comment:
Inlining these two fields will make the tests that use them more readable. (Same on other tests)

Reply:
but DRY ? :)  will change as suggested.

Reply:
I agree with DRY in production code. Sometimes factoring such things out in tests just makes the test really hard to grok.